### ISSUE TITLE
Update ReadMe to remove Tiller

### ISSUE THREAD

**Issue by gozie11** [Spazzy757/helm-to-kustomize#3]
Hi I'm trying to follow the guide in your readme to template out some kustomize resource via helm chart, but am confused as to whether or not the section about Tiller's issues with namespaces is still relevant.

**Comment by Spazzy757** [comment]
you're correct, Tiller is no longer part of Helm, but helm still does not handle namespace creation well
